extension/src/experiments/columns/model.ts

```diff
@@ -63,6 +73,14 @@ export class ColumnsModel extends PathSelectionModel<Column> {
     )
   }
 
+  public setColumnHeadLayersMax(max: number) {
+    this.columnHeadLayersMaxLength = max
+    this.persist(
```

Comment:
Rather than store this in the memento you might want to set a config option. Take a look at `extension/src/vscode/config.ts` as an entry point for the extension's existing config options.